src/main.cpp

```diff
@@ -195,6 +200,9 @@ int main(int argc, char **argv) {
     case 'g':
       garbage_collection_period = (time_t)atoi(optarg);
       break;
+    case 'P':
+      pidsToWatch.insert((pid_t) atoi(optarg));
```

Comment:
Should I really check for negative numbers(since, pids are always positive)?  
Should I check for non zero pid? (User can provide 0, or, an invalid string, which, **atoi** will return zero then).

Reply:
OK by me